modules/patterns/chessboard.js

```diff
@@ -0,0 +1,35 @@
+import { Material } from '../material.js';
+
+export class Chessboard extends Material {
+    constructor(color1, color2, size = 1) {
+        super();
+        this.color1 = color1;
+        this.color2 = color1;
```

Comment:
Could be a misspelling here ? 
We have the `color2` in constructor argument and still assign `color1` to `this.color2`